setup.py

```diff
@@ -28,7 +28,7 @@
     # Versions should comply with PEP440.  For a discussion on single-sourcing
     # the version across setup.py and the project code, see
     # https://packaging.python.org/en/latest/single_source_version.html
-    version='1.8.2',
+    version='1.8.3',
 
```

Comment:
should we bump the version to 1.9.1? Since new feature is added? not minor?

Reply:
I've bumped to 1.9.0